calibrators/tag_based_pnp_calibrator/README.md

```diff
@@ -0,0 +1,123 @@
+| `lidartag_measurement_noise_translation`      | `double`              | `0.05`                           | The square of this value in meters is part of the input for the measurement noise covariance matrix (R) in the Kalman filter.                                                                                                                                           |
+| `lidartag_measurement_noise_rotation`         | `double`              | `5.0`                            | The square of this value in degrees is part of the input for the measurement noise covariance matrix (R) in the Kalman filter.                                                                                                                                          |
+| `lidartag_process_noise_translation`          | `double`              | `0.01`                           | The square of this value in meters is part of the input for the process noise covariance matrix (Q) in the Kalman filter.                                                                                                                                               |
+| `lidartag_process_noise_translation_dot`      | `double`              | `0.001`                          | The square of this value in meter/second is part of the input for the process noise covariance matrix (Q) in the Kalman filter.                                                                                                                                         |
+| `lidartag_process_noise_rotation`             | `double`              | `1.0`                            | The square of this value in degrees is part of the input for the process noise covariance matrix (Q) in the Kalman filter.                                                                                                                                              |
+| `lidartag_process_noise_rotation_dot`         | `double`              | `0.1`                            | The square of this value in degree/second is part of the input for the process noise covariance matrix (Q) in the Kalman filter.                                                                                                                                        |
+| `apriltag_max_convergence_translation`        | `double`              | `2.0`                            | Maximum Threshold in pixels for corners Kalman filter's covariance matrix to consider the hypothesis as converged.                                                                                                                                                      |
```

Comment:
Why `T`hreshold?

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/dcdf2ed43c8f84dc15204dd35f03bdc09740762b